Sources/NetShears/URLProtocol/NetworkIgnore.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  NetworkIgnore.swift
+//  
+//
+//  Created by Mehrdad Goodarzi(Arash) on 6/24/22.
+//
+
+import Foundation
+
+public enum Ignore {
+    case disbaled
+    case enabled(ignoreHandler: ((URLRequest) -> Bool)?, grpcIgnoreHandler: ((NetShearsRequestModel) -> Bool)?, onFeatures: [IgnoringFeature])
```

Comment:
Wouldn't it be better to add the ignore handler as a delegate method? In my opinion disabled option won't be useful that much (as it ignores all the traffic). Something like this:
```swift
protocol IgnoreDelegate {
  func shouldIgnore(request: NetShearsRequestModel) -> Bool
}

NetShears {
  weak var ignoreDelegate: IgnoreDelegate
}

// usage
NetShears.shared.ignoreDelegate = ...
```

Reply:
approach changed. please resolve outdated comments